doc/changes/latest.inc

```diff
@@ -62,7 +62,7 @@ Bugs
 - Allow int-like for the argument ``id`` of `~mne.make_fixed_length_events` (:gh:`11748` by `Mathieu Scheltienne`_)
 - Fix bugs with :mod:`mne.decoding` classes (:gh:`11846` by `Eric Larson`_):
   - :class:`mne.decoding.ReceptiveField` now sets ``rf.fit_intercept_`` during ``fit()`` rather than overwriting ``rf.fit_intercept``
-  - :class:`mne.decoding.SlidingEstimator` now returns the same ``dtype`` as ``X`` (usually ``float64``) in ``sl.predict(X)`` rather than always returning ``int``
+  - :class:`mne.decoding.SlidingEstimator` now returns the same ``dtype`` as ``X`` (usually ``float64``) in ``sl.transform(X)`` rather than always returning ``int``
```

Comment:
```suggestion
  - :class:`mne.decoding.SlidingEstimator` now returns the same ``dtype`` as ``X`` (usually ``float64``) in ``sl.transform(X)`` rather than always returning ``int`` (:gh:`11846` by `Eric Larson`_)
```